iceoryx_platform/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx_platform</name>
+    <version>2.90.0</version>
+    <description>Eclipse iceoryx inter-process-communication (IPC) middleware platform</description>
+    <maintainer email="iceoryx-oss-support@apex.ai">Eclipse Foundation, Inc.</maintainer>
+    <license>Apache 2.0</license>
+    <url type="website">https://iceoryx.io</url>
+    <url type="bugtracker">https://github.com/eclipse-iceoryx/iceoryx/issues</url>
+    <url type="repository">https://github.com/eclipse-iceoryx/iceoryx</url>
+
+    <buildtool_depend>cmake</buildtool_depend>
+
+    <depend>acl</depend>
+    <depend>libatomic</depend>
+
+    <doc_depend>doxygen</doc_depend>
+
+    <export>
+        <build_type>cmake</build_type>
+    </export>
+</package>
```

Comment:
Shouldn't this also be part of the specific platform itself?

Reply:
In my opinion no. Every platform looks identical only the implementation differs and for me the platform is the uniform access to every underlying platform therefore this is the header for the platform library.

For me it is like `iceoryx_posh` has also not a different package name only because it was compiled on windows

Reply:
Yes, but not every platform has `acl` or has it?

Reply:
Yes, every platform has `acl` but on some it is working and on some it is just a dummy. But every platform has the potential to introduce an `acl` like behavior with its tools.

Reply:
Maybe I'm wrong, but this is not about introducing `acl` like behavior but requiring a package called `acl`. On the other side it worked like that for years so it should be okay ... I hope this sentence does not result in an immediate CI failure on some platforms :laughing: